src/screens/LoginScreen/LoginScreen.tsx

```diff
@@ -1,61 +1,78 @@
-import React, { useState } from 'react';
-import { View, Alert } from 'react-native';
-import { TextInput } from 'react-native-paper';
-import login from '../../util/login';
-import { Title, LinkButton, SpacerInline } from '../../elements';
-import InputLabel from '../../elements/FormTextInput/InputLabel';
-import styles from './LoginScreen.styles';
-import { useNavigation, useNavigationParam } from 'react-navigation-hooks';
+import React, { useState } from "react";
```

Comment:
Please leave in the single quotes here, double quotes (outside of React components) are a signal that the string contains a single quote.  I'm working on a linting setup that will make these guidelines more obvious in the future.